src/Generators/CsvDictionary.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Gettext\Generators;
+
+use Gettext\Translations;
+
+class CsvDictionary extends Generator implements GeneratorInterface
+{
+    /**
+     * {@parentDoc}.
+     */
+    public static function toString(Translations $translations)
+    {
+        $array = PhpArray::toArray($translations);
+
+        //for a simple json translation dictionary, one domain is supported
+        $values = current($array);
+
+        // remove meta / header data
+        if (array_key_exists('', $values)) {
+            unset($values['']);
+        }
+
+        $tmpFile = tempnam(sys_get_temp_dir(), 'gettext_');
+        $handle = fopen($tmpFile, 'w');
```

Comment:
Why not use ` fopen('php://temp', 'w') `? Seems more optimized than creating a real file in disk.